src/operator/numpy/np_elemwise_binary_op.cc

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file np_elemwise_binary_op.cc
+ * \brief CPU Implementation of numpy-compatible bitwise_xor operators
+ */
+
+#include <mxnet/base.h>
+#include "../mshadow_op.h"  // mshadow operations
+#include "../operator_common.h"  // MakeZeroGradNodes
+#include "../tensor/elemwise_binary_op.h"  // ElemwiseShape, ElemwiseType
+#include "../tensor/elemwise_binary_broadcast_op.h"  // BinaryBroadcastCompute
+
+namespace mxnet {
+namespace op {
+
+NNVM_REGISTER_OP(_np_bitwise_xor)
+.set_num_inputs(2)
+.set_num_outputs(1)
+.set_attr<mxnet::FInferShape>("FInferShape", BinaryBroadcastShape)
+.set_attr<nnvm::FInferType>("FInferType", ElemwiseType<2, 1>)
```

Comment:
Did you check the input is of type boolean or integer? 
Numpy only accept those two input types

Reply:
Only integer type is handled. boolean type not handle. i have wrote  it in comments

Reply:
You need to print out correct error message when the dtype is not supported. Please add a specialized function for this.

Reply:
done

Reply:
resolved, it's my first op, i donot use npx.set_np(), and i attempt again, in mxnet, bool will be changed  to 1.0 or 0.0, so it can support bool.